charts/country-risk-backend-charts/README.md

```diff
@@ -14,7 +14,7 @@ helm install release_name ./charts/country-risk-backend-charts --namespace your_
 ```
 
 This will install a new release of the Country Risk in the given namespace.
-On default values this release deploys the latest image tagged as `v1.0.0` from the repository's GitHub Container Registry.
+On default values this release deploys the latest image tagged as `"v1.0.4"` from the repository's GitHub Container Registry.
```

Comment:
Like in the workflows, try to avoid referencing specific versions, since they change too frequently. In this case reference the 'AppVersion', and your text will always be valid

Reply:
Changed also to a not hardcoded version and pointed to the .chart.AppVersion